spec/system/page_objects/pages/category.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    class Category < PageObjects::Pages::Base
+      # keeping the various category related features combined for now
+
+      def visit(category)
```

Comment:
We were talking with martin of doing this kind of for various resources, I did on frontend for channels in chat, and its quite handy